chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ClientsDataSource.swift
+//  chargeOclock
+//
+//  Created by Goran Blažič on 08/01/2017.
+//  Copyright © 2017 goranche.net. All rights reserved.
+//
+
+import UIKit
+
+class ClientsDataSource: NSObject, UITableViewDataSource {
+
+	let clients: [Client]
+	let reuseIdentifier: String
```

Comment:
I know it's little early for things like this. But how will we handle access levels in this project? Does reuseIdentifier for instance really need to be internal?